container/image.bzl

```diff
@@ -380,6 +394,13 @@ _attrs = dict(_layer.attrs.items() + {
         executable = True,
         allow_files = True,
     ),
+    # null_cmd and null_entrypoint are hidden attributes from users.
```

Comment:
You need to document somewhere much more visible that explicitly setting cmd or entrypoint to None, [] or "" will have this behavior (here: https://github.com/bazelbuild/rules_docker#container_image-1 , https://github.com/bazelbuild/rules_docker/blob/master/container/image.bzl#L476,  https://github.com/bazelbuild/rules_docker/blob/master/container/image.bzl#L482)

Reply:
Done.